packages/@react-spectrum/dialog/test/DialogTrigger.test.js

```diff
@@ -136,6 +136,84 @@ describe('DialogTrigger', function () {
     expect(popover).toHaveAttribute('style');
   });
 
+  describe('popover arrows', function () {
+    const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight');
+    const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth');
+
+    beforeAll(() => {
+      Object.defineProperty(HTMLElement.prototype, 'offsetHeight', {configurable: true, value: 500});
```

Comment:
I think you want to use jest.spyOn
This way they will be known by jest as mocks and will get restored automatically when the test is done